shared/color.js

```diff
@@ -1,4 +1,6 @@
-export const whiteTheme = "#EAEFF2";
-export const greenTheme = "#23B0A3";
-export const darkGreenTheme = "#0c9186";
-export const obsidianTheme = "#4C5270";
+export const colors = {
+  whiteTheme: "#EAEFF2",
```

Comment:
As I said, color is not a theme, so please don't call them "Theme". Just "white" or "green" is enough.